sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -59,7 +59,7 @@ public TextAnalyticsClient(Uri endpoint, TokenCredential credential, TextAnalyti
         {
             Argument.AssertNotNull(endpoint, nameof(endpoint));
             Argument.AssertNotNull(credential, nameof(credential));
-            options ??= new TextAnalyticsClientOptions();
+            Argument.AssertNotNull(options, nameof(options));
```

Comment:
What's the thinking behind this change?  I believe the prior approach is precedented in other libraries.

Reply:
I was thinking that if the user doesn't want/need to pass the ClientOptions, then they should just use [this ](https://github.com/Azure/azure-sdk-for-net/pull/9486/files#diff-9c32af2358a2a0fc85cdfe85da0d7eaaL43) constructor, instead of sending null here.

Reply:
Yep, makes sense.  I was looking at KeyVault, which passes in null in the other constructor (https://github.com/Azure/azure-sdk-for-net/blob/40a83baae05bb095b98bd319e539772420c57da9/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClient.cs#L54).  I'm fine with this approach too.